templates/CRM/Admin/Form/Setting/Miscellaneous.tpl

```diff
@@ -52,7 +52,7 @@
       <tr class="crm-miscellaneous-form-block-wkhtmltopdfPath">
         <td class="label">{$form.wkhtmltopdfPath.label}</td>
         <td>{$form.wkhtmltopdfPath.html}<br />
-          <p class="description">{ts 1="http://wkhtmltopdf.org/"}<a href="%1">wkhtmltopdf is an alternative utility for generating PDF's</a> which may provide better performance especially if you are generating a large number of PDF letters or receipts. Your system administrator will need to download and install this utility, and enter the executable path here.{/ts}</p>
+          <p class="description">{assign var="wkhtmltopdf_link" value='<a href="https://wkhtmltopdf.org/" target="_blank">wkhtmltopdf</a>'}{ts 1=$wkhtmltopdf_link}%1 is an alternative utility for generating PDF's which may provide better performance especially if you are generating a large number of PDF letters or receipts. Your system administrator will need to download and install this utility, and enter the executable path here.{/ts}</p>
```

Comment:
Nitpick: I would avoid this change. It was working OK, and it will break translations. In general we use `<a href="%1">` in ts strings (where anything extra is in the %1).

Reply:
@mlutfy don't you mean `<a %1> ` instead of `<a href="%1">`?
I prefer the first over the second, because with the first you can add other attributes (class, target, title) without changing the string in gettext.

Reply:
yep, I was mixed up. My other comments below used that syntax.